R/dominantTaxa.R

```diff
@@ -174,3 +233,62 @@ setMethod("addPerSampleDominantTaxa", signature = c(x = "SummarizedExperiment"),
             addPerSampleDominantFeatures(x, ...)
         }
 )
+
+
+########################## HELP FUNCTIONS summary ##############################
+
+# top entries in a vector or given field in a data frame
+# from microbiome package
+
+top <- function (x, field = NULL, n = NULL, output = "vector", round = NULL, na.rm = FALSE, include.rank = FALSE) {
```

Comment:
Run BiocCheck::BiocCheck() this line seems to be too long

Also, we have not used spaces after function -->

```
name <- function(x, y, ...){
}
```